### PR TITLE
Add general information on internet disabled mode

### PR DESCRIPTION
**Pull request by rosemm**
Putting this in as a PR to make it easier for other teams (PM, Outreach, etc.) to review content in draft. 
Here's a link to see the new page rendered in liascript:

[Lab Security](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/internet_disabled_general/arcus_orientation.md#27)

### CODE REVIEW

File: arcus_orientation.md [hunk 1]
```diff
@@ -694,9 +694,32 @@ The [training videos](#training-videos) walk through everything you need to get
...
+
+Using Internet Disabled Mode helps protect you from both of these risks, by allowing you to work on your code in a space that is entirely disconnected from the internet. 
+
+But how do I install packages without internet?
+------
+Of course, you'll still need to be able to install packages to do your work. For this, we've developed the **Package Installation Form**. Rather than installing your packages in the same environment that includes your data, filling out this form instantly installs the necessary packages in a separate environment that doesn't have any data in it. This ensures that, even if there is any malicious code in the package you've installed, it doesn't have any access to your data when it executes. The package is then immediately made available for your use in your Internet Disabled lab.
```
Comment (murphyke) [inline arcus_orientation.md:715]: * "instantly" may not be the exact right wording here. The intent is to to convey that no approval process is required, but the package installation can be expected to take a bit longer than using conventional installation approaches.
* You probably want to mention the download capability of the install form. This additional use case (in addition to brevity) is why we internally gravitated to the moniker "Install Form".
* Re: malicious code, there are two aspects. 1) During installation itself, if the package _installation scripts_ (as opposed to the package itself) are malicious, then there is no access to data. 2) When the package is actually used in the lab (e.g. run as part of an R or Python analysis program), if the package is malicious, the package will have access to the data, but it will at least not be able to "phone home" to a malicious server to exfiltrate data or download additional malicious software. I know it's hard to convey the nuance here.

File: arcus_orientation.md [hunk 2]
```diff
@@ -694,9 +694,32 @@ The [training videos](#training-videos) walk through everything you need to get
...
+
+But how do I install packages without internet?
+------
+Of course, you'll still need to be able to install packages to do your work. For this, we've developed the **Package Installation Form**. Rather than installing your packages in the same environment that includes your data, filling out this form instantly installs the necessary packages in a separate environment that doesn't have any data in it. This ensures that, even if there is any malicious code in the package you've installed, it doesn't have any access to your data when it executes. The package is then immediately made available for your use in your Internet Disabled lab. 
+
+When should I use Internet Disabled Mode?
+---
+Since its not always easy to tell when legitimate packages may be accessing the internet, **we recommend always using Internet Disabled mode**, not just when installing packages. That being said, it is still possible to enable access to the internet if needed, such as during particularly complex package installations. (Note that it is possible to allow Internet Enabled mode only for certain users of a lab.)
```
Comment (murphyke) [inline arcus_orientation.md:719]: Re: allowing only for certain users, it might be more clear to mention that the PI of a project is able to specify that some lab users never have Internet access.

File: arcus_orientation.md (hunk 1 above)
Comment (franzenr) [reply arcus_orientation.md:715]: Thanks for these comments, Kevin! I'll try to come up with some adjusted wording -- to better convey the nuance without being too wordy!

File: arcus_orientation.md (hunk 2 above)
Comment (franzenr) [inline arcus_orientation.md:719]:
```suggestion
Since its not always easy to tell when legitimate packages may be accessing the internet, **we recommend always using Internet Disabled mode**, not just when installing packages. That being said, it is still possible to enable access to the internet if needed, such as during particularly complex package installations. (Note that the PI is able to completely restrict Internet access for some users. In those cases, it is not possible for those users to enable Internet access.)
```

File: arcus_orientation.md [hunk 3]
```diff
@@ -694,9 +694,41 @@ The [training videos](#training-videos) walk through everything you need to get
...
+
+Using Internet Disabled Mode helps protect you from both of these risks, by allowing you to work on your code in a space that is entirely disconnected from the internet. 
+
+But how do I install packages without internet?
+------
+Of course, you'll still need to be able to install packages to do your work. For this, we've developed the **Package Installation Form**. Rather than installing your packages in the same environment that includes your data, filling out this form instantly installs the necessary packages in a separate environment that doesn't have any data in it. This ensures that, even if there is any malicious code in the package you've installed, it doesn't have any access to your data when it executes. The package is then immediately made available for your use in your Internet Disabled lab.
```
Comment (franzenr) [inline arcus_orientation.md:715]:
```suggestion
Of course, you'll still need to be able to install packages or download files to do your work. For this, we've developed the **Installation Form**. Rather than installing your packages in the same environment that includes your data, when you fill out this form, the packages automatically begin installing in a separate environment that doesn't have any data in it.This ensures that, if there is any malicious code in the scripts that govern the installation process, it does not have any access to your data while it executes. Once the packages have finished installing, they become available to you in your lab. At this point, by subsequently running your code in your lab in Internet Disabled mode, even if there is any malicious code in the package you've installed, it doesn't have any access to the Internet in order to exfiltrate your data.
```

File: arcus_orientation.md [hunk 4]
```diff
@@ -694,9 +694,41 @@ The [training videos](#training-videos) walk through everything you need to get
 
 ***
 
-## Lab Security
+## Security Feature: Internet Disabled Mode
 
-(here's where general content about why we disable internet goes)
+While there are myriad benefits to taking a scripted approach to data analysis, it does also introduce some specific privacy and security risks. Thankfully, Arcus has taken these risks into consideration, and developed our [Internet Disabled Mode](https://forum.arcus.chop.edu/t/about-the-arcus-lab-security-setting-for-disabling-internet-in-labs/741) feature. The feature provides peace of mind by allowing you to work on your code in an environment that is entirely disconnected from the internet, keeping your data and code safer.
```
Comment (Tavo4201) [inline arcus_orientation.md:699]: Izzi-F9EY
